cmd/geth/testdata/vcheck/vulnerabilities.json

```diff
@@ -52,19 +52,68 @@
     "check": "Geth\\/v1\\.9\\.(7|8|9|10|11|12|13|14|15|16).*$"
   },
   {
-    "name": "GethCrash",
+    "name": "Geth DoS via MULMOD",
```

Comment:
this whole file needs to be reviewed. If not planning to do so for this PR, then create an issue about this.